### DISCUSSION TITLE
How to get metadata via mdx.macro for create-react-app?

### DISCUSSION THREAD

**Discussion by lefnire** [mdx-js/mdx#1362] [Q&A] [answered]
The [create-react-app](https://mdxjs.com/getting-started/create-react-app) sample uses the following:

```javascript
import {importMDX} from 'mdx.macro'
const Content = lazy(() => importMDX('./Content.mdx'))
```
With `const Content =` pulling out the component to be rendered, how to I get `metadata` as per the [other examples](https://mdxjs.com/getting-started/) `import MDXDocument, {metadata} from 'posts/post.mdx'`?

**Comment by ChristianMurphy** [comment, marked as answer]
See https://github.com/mdx-js/mdx/discussions/1355#discussioncomment-148993
which shows how to use the mdx webpack loader with CRA, to get the same results as https://mdxjs.com/getting-started#exports
